README.md

```diff
@@ -1,21 +1,61 @@
 # scalar-admin
 
-scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications.
+scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications. 
+It will help you to `PAUSE` and `UNPAUSE` the ledger to create point-in-time recovery (PITR). 
 
-## Use the client-side tool
+## Client-side tool
+
+The scalar-admin client tool is used to manage the admin interface implemented applications.
+It will help to `PAUSE` and `UNPAUSE` the application without losing the transaction.
+
+### Installation
+
+You can create a build for the scalar-admin client tool using the following methods
+
+#### Jar
+
+If you want to create a client tool fat jar, you can do it as follows.
 
-To access scalar-admin integrated applications, you can use the fat jar `scalar-admin-<version>-all.jar` found in [releases](https://github.com/scalar-labs/scalar-admin/releases) as follows.
 ```console
-$ java -jar scalar-admin-<version>-all.jar
+$ ./gradlew shadowJar
 ```
 
-You can also build the tool as follows.
+#### Container
+
+If you want to create a scalar-admin container, you can do it as follows.
 
 ```console
-$ ./gradlew installDist
-$ build/install/scalar-admin/bin/scalar-admin
+$ ./gradlew docker
 ```
 
+### Usage
+
+You can manage the scalar-admin integrated applications in the following ways
+
+#### Jar
+
+To access scalar-admin integrated applications, you can use the fat jar `scalar-admin-<version>-all.jar` found in [releases](https://github.com/scalar-labs/scalar-admin/releases) as follows
+
+```console
+$ java -jar scalar-admin-<version>-all.jar -c <Command> -s <SRV_Service_URL>
```

Comment:
Is any details regarding `<SRV_Service_URL>` necessary in this guide, like in notes section?